js/server/modules/org/arangodb/foxx/request_context.js

```diff
@@ -448,6 +465,10 @@ extend(RequestContext.prototype, {
 /// and accordingly the return value of the *params* for the body call will also
 /// return an array of models.
 ///
+/// Alternatively you can provide a joi schema instead of a model to allow
+/// arbitrary data. When using a joi schema or a model that has a joi schema,
```

Comment:
It allows strings, numbers and other things. Of course it only allows well-formed JSON, but that's obvious. Should we mention that bodyParam only works with JSON request bodies? I thought that's a given.

Also, there's `joi.any()`, which explicitly allows anything.